tests/interaction/line_edge_intersection.cc

```diff
@@ -0,0 +1,85 @@
+#include <fiddle/interaction/interaction_utilities.h>
+
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/fe/fe_nothing.h>
+#include <deal.II/fe/mapping_q.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/tria.h>
+
+#include <ibtk/AppInitializer.h>
+#include <ibtk/IBTKInit.h>
+
+using namespace dealii;
+using namespace SAMRAI;
+
+// Test the line edge intersection code
+
+void
+test(SAMRAI::tbox::Pointer<IBTK::AppInitializer> app_initializer)
+{
+  double d_stencil_center_x = 0.0;
```

Comment:
The `d_` prefix in SAMRAI/IBAMR is only for member variables so we shouldn't use it here. For all of these scalar values we should just use `getDoubleWithDefault()` or an equivalent.